### PR TITLE
Add missing links in README.md

### PR DESCRIPTION
**Pull request by totorigolo**
Adds two missing links in the README, to the Guides and the API documentation.

Fixes: #1229

### CODE REVIEW

**Comment by atouchet** [comment]
Maybe the URLs should be updated: https://github.com/tokio-rs/tokio/issues/1229#issuecomment-507854891

File: README.md [hunk 1]
```diff
@@ -107,6 +107,8 @@ First, see if the answer to your question can be found in the [Guides] or the
 the [Tokio Gitter channel][chat]. We would be happy to try to answer your
 question.  Last, if that doesn't work, try opening an [issue] with the question.
 
+[Guides]: https://tokio.rs/docs/getting-started/hello-world/
```
Comment (carllerche) [inline README.md]:
```suggestion
[Guides]: https://tokio.rs/docs
```

/docs should redirect as needed.

**Comment by carllerche** [comment]
> Maybe the URLs should be updated: [#1229 (comment)](https://github.com/tokio-rs/tokio/issues/1229#issuecomment-507854891)

I'm :+1: on including the exact version in the `docs.rs` link as the `README` is included in crates being released.

File: README.md (hunk 1 above)
Comment (totorigolo) [reply README.md]: Should we also update the other links to the Guides?

```
$ rg -n 'tokio.rs/docs/getting-started/hello-world'
README.md
30:[Guides](https://tokio.rs/docs/getting-started/hello-world/) |
110: the suggested change

tokio/README.md
30:[Guides](https://tokio.rs/docs/getting-started/hello-world/) |

tokio/src/lib.rs
24://! [website]: https://tokio.rs/docs/getting-started/hello-world/
```

Comment (carllerche) [reply README.md]: Probably! `/docs` should be more reliable as an entry point.

**Comment by atouchet** [comment]
Also should the doc links in the Readme be updated to 0.1.22? It looks like those haven't been kept up to date.

**Comment by carllerche** [comment]
@atouchet Yeah... all this manual work would ideally be automated...

**Comment by totorigolo** [comment]
Updated to `0.1.22` and shortened links to `docs/`.

> @atouchet Yeah... all this manual work would ideally be automated...

I can write a small Bash script to do that, if you want and think that it could be useful (and don't see any security issue in executing a tracked Bash script).

**Comment by carllerche** [comment]
@totorigolo could be helpful, especially if it was based on the version in `Cargo.toml` :+1:

**Pull request merged**